src/secrets/components/SecretContextMenu.tsx

```diff
@@ -42,19 +44,35 @@ const SecretContextMenu: FC<Props> = ({secret}) => {
     }
   }
 
+  // Note to self: add a class on the context element to push everything left about
+  // 20 pixels because there's no padding on the rightmost button for reasons
   return (
-    <Context>
+    <Context className="secrets-context-menu">
       <CopyToClipboard text={secret.id} onCopy={handleCopyAttempt}>
         <Button
           testID={`copy-to-clipboard--${secret.id}`}
           size={ComponentSize.Small}
           color={ComponentColor.Secondary}
           text="Copy to Clipboard"
-          className="secret-context-menu--copy-to-clipboard"
           shape={ButtonShape.StretchToFit}
           style={{margin: '4px 0 0 0'}}
         />
       </CopyToClipboard>
+      <Context.Menu
+        testID={`delete-secret-initial--${secret.id}`}
+        size={ComponentSize.Small}
+        color={ComponentColor.Danger}
+        icon={IconFont.Trash}
+        text="Delete"
+        shape={ButtonShape.StretchToFit}
+        style={{margin: '4px 0 0 10px'}}
+      >
+        <Context.Item
+          label="Delete"
+          action={onDeleteSecret}
```

Comment:
nit: we can probably just move the delete functionality here rather than just prop drilling the dispatched function

Reply:
Moved.